.github/workflows/ci.yml

```diff
@@ -145,7 +147,7 @@ jobs:
           key: yarn-${{ hashFiles('yarn.lock') }}
       - name: Install
         run: |
-          BABEL_ENV=test-legacy make -j bootstrap-only
+          make -j bootstrap-only
```

Comment:
```suggestion
          yarn install
```

Reply:
Oh ok, if we are just installing without building then yeah unnecessary - do we not need to build in this case?